doc/classes/CubeMap.xml

```diff
@@ -45,6 +49,7 @@
 			<return type="int">
 			</return>
 			<description>
+				Returns the width of the CubeMap.
```

Comment:
- Per the writing guidelines, use possessive:

Returns the CubeMap's width.

Reply:
Got those, thanks :)